content/changes.md

```diff
@@ -8,6 +8,28 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## March 31, 2024  
+
+### Improvements 
+
+- **Added preloaded examples** 
+
+  Add preloaded example for Rust and Go 
+
+- **Directory To Tree: allow async function** 
```

Comment:
Too minor, I'd remove. It's a small-ish content adjustment and we haven't documented these in the past. 99% of our customers aren't using this challenge or don't care about it.